sample.tex

```diff
@@ -54,12 +54,18 @@
 
 % graphics packages
 \usepackage[pdftex]{graphicx} % remove pdftex if you are not compiling to pdf
-\graphicspath{{./figures/}} % this places all graphics in the figures subdirectory
+\graphicspath{{./figures/}{../figures/}} % this places all graphics in the figures subdirectory
```

Comment:
why are we adding `../figures/`?  I'm inclined not to do this (unless there is a reason I'm missing) because it will potentially include figures from a directory that is outside of this repo, which may cause some confusion.

Reply:
Unfortunately, this is another artifact of using `subfiles` across directories. When compiling components individually, the relative path is traced from the subfile and not from the main file.